packages/i18nify-js/src/modules/geo/getCities.ts

```diff
@@ -0,0 +1,53 @@
+import { withErrorBoundary } from '../../common/errorBoundary';
+import {
+  I18NIFY_DATA_SOURCE,
```

Comment:
same as above.

Why can't we directly use `#/i18nify-data/country/subdivisions/IN.json` ?